src/FileStorageServiceProvider.php

```diff
@@ -18,7 +18,12 @@ public function register(Container $container): void
             $configParams = $config['config'] ?? [];
             $aliases = $config['aliases'] ?? [];
             $adapter = $factory->create($config['adapter']);
-            $container->set($alias, new Filesystem($adapter, $aliases, $configParams));
+            $container->set($alias, [
```

Comment:
I think it's better to make a closure here

Reply:
May you wrote about the old code, it is an array definition now.

Reply:
Done